Tests/Svg.UnitTests/ImageComparisonTest.cs

```diff
@@ -28,6 +28,9 @@ public class ImageComparisonTest
         [TestCaseSource(typeof(ImageTestDataSource), nameof(ImageTestDataSource.PassingTests))]
         public void CompareSvgImageWithReference(ImageTestDataSource.TestData testData)
         {
+            // W3C Test Suites use external references to local fonts
+            SvgDocument.ResolveExternalResources = true;

```

Comment:
Is it possible to move this line to head of `CompareSvgImageWithReferenceImpl` method ?

Reply:
I did so in an earlier commit, but moved it back here because I now also use `CompareSvgImageWithReferenceImpl` from `XxeVulnerabilityTests`, which need `XxeVulnerabilityTests` to not be overwritten. I've added a few suggestions for alternative organization of the test here https://github.com/svg-net/SVG/pull/873#issuecomment-841139774.

I can also make the value for `ResolveExternalResources` a parameter of `CompareSvgImageWithReferenceImpl` and pass it in from `CompareSvgImageWithReference` and `XxeResolveExternalImage`? However you'd prefer.

Reply:
In `XxeVulnerabilityTests`, please evaluate DOM value etc. instead of comparing the images.

Now in `ImageComparisonTest`, I think that I do is only in case of `SvgDocument.ResolveExternalResources = true`.

Reply:
For both `<image xlink:href="..." />` and `<tref xlink:href="..."/>`, resolving the external reference only seems to happen during rendering, and even after calling `Draw`, I can't see anything that has changed in the DOM which a test could look for. Am I overlooking something?

Reply:
Please add property or method to evaluate.